### PR TITLE
fix(dist): generate doc & exit script when upload files failed

### PR DESCRIPTION
**Pull request by imbajin**
NOTE: DON'T DELETE THIS BRANCH (afer merge it)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -279,7 +276,7 @@
 
     <profiles>
         <profile>
-            <id>release</id>
+            <id>apache-release</id>
```
Comment (imbajin) [inline pom.xml:279]: @coderzc @zyxxoo @z7658329 
seems we should use `apache-release` id here to override the settings when execute `mvn xx -Papache-release`

File: hugegraph-dist/scripts/apache-release.sh [hunk 2]
```diff
@@ -92,9 +95,10 @@ svn status
 
 ##### 4.3 commit & push files
 if [ "$USERNAME" = "" ]; then
-  svn commit -m "submit files for ${REPO} ${RELEASE_VERSION}"
+  svn commit -m "submit files for ${REPO} ${RELEASE_VERSION}" || exit
```
Comment (javeme) [inline hugegraph-dist/scripts/apache-release.sh:98]: prefer to add "set -ev" in the file header

Comment (imbajin) [reply hugegraph-dist/scripts/apache-release.sh:98]: It was added before, but the output content is **too much** and makes a little messy

consider test & add it again

Comment (javeme) [reply hugegraph-dist/scripts/apache-release.sh:98]: ok

**Review by javeme — APPROVED** [review]

**Comment by imbajin** [comment]
rebase & merge this PR, so as other `release-1.0.0 -> master`

**Review by zyxxoo — APPROVED** [review]

**Review by javeme — APPROVED** [review]

**Pull request merged**